tests/helpers/index.ts

```diff
@@ -33,3 +36,14 @@ export function currentURL() {
     const router = owner.lookup('router:main');
     return router.get('location').getURL();
 }
+
+export function setupOSFApplicationTest(hooks: any) {
+    setupApplicationTest(hooks);
+    hooks.beforeEach(async function(this: TestContext) {
+        faker.seed(17);
```

Comment:
It kinda feels wrong to have the same seed for every test... but I can't come up with a good reason why, so I'm probably worrying about nothing.

Reply:
I mean, it’s definitely not random.